scripts/main.js

```diff
@@ -71,4 +71,83 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+// ========================== OPDRACHT 1A ========================== //
+console.log('Opdracht 1a');
+console.log(`De afdeling Sales heeft ${departments.sales.numberOfEmployees} medewerkers`);
+console.log();
+// ========================== OPDRACHT 1B ========================== //
+console.log('Opdracht 1b');
+console.log(`Marketing is een leuke afdeling om te werken. ${departments.marketing.description}.`);
+console.log();
+// ========================== OPDRACHT 1C ========================== //
+console.log('Opdracht 1c');
+console.log(`De afdeling Customer Service heeft ${departments["customer-service"].numberOfEmployees} medewerkers`);
+console.log();
+// ========================== OPDRACHT 1D ========================== //
+console.log('Opdracht 1d');
+console.log(`Sales is een uitdagende afdeling om te werken als "Verkoopmanager". ${departments.sales.jobs[1].description}`);
+console.log();
+console.log();
+
+
+// ========================== OPDRACHT 2A, 2B en 2C ========================== //
+console.log('Opdracht 2a, 2b en 2c');
+const departmentChoice = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
+
+switch (departmentChoice) {
+    case 'marketing':
+        // console.log(`Je koos ${departmentChoice}. ${departments.marketing.description}`);
+        console.log(`${departmentChoice.toUpperCase()} is een leuke afdeling om te werken. Er werken op dit moment ${departments[departmentChoice].numberOfEmployees} medewerkers.`)
+        break;
+    case 'sales':
+        // console.log(`Je koos ${departmentChoice}. ${departments.sales.description}`);
+        console.log(`${departmentChoice.toUpperCase()} is een leuke afdeling om te werken. Er werken op dit moment ${departments[departmentChoice].numberOfEmployees} medewerkers.`)
+        break;
+    case 'customer-service':
+        // console.log(`Je koos ${departmentChoice}. ${departments["customer-service"].description}`);
+        console.log(`${departmentChoice.toUpperCase()} is een leuke afdeling om te werken. Er werken op dit moment ${departments[departmentChoice].numberOfEmployees} medewerkers.`)
+        break;
+    default:
+        console.error('Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.');
+        document.getElementById('error-message').textContent = `Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.`;
+}
+
+// ========================== OPDRACHT 3A, 3B en 3C ========================== //
+// console.log('Opdracht 3a, 3b en 3c');
+
+// ========================== OPDRACHT 4A, 4B en 4C ========================== //
+
+const jobChoice = prompt(`Je koos ${departmentChoice.toUpperCase()}. Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in.
```

Comment:
De reden dat je een error krijgt is omdat `jobChoice` de waarde `"2"` bijv. heeft in plaats van `2`. Daardoor kom je in de default van de onderstaande switch terecht. 

Wat je kunt doen is `const jobChoiceNumber = Number(jobChoice);`. Vervolgens kun je die waarde dan weer in de onderstaande switch stoppen.